Code/GraphMol/SmilesParse/CXSmilesOps.cpp

```diff
@@ -1300,6 +1300,19 @@ std::string quote_string(const std::string &txt) {
   return txt;
 }
 
+std::string quote_atomprop_string(const std::string &txt) {
```

Comment:
It appears that quoting is different based on what you are quoting, so making a new function made sense.